packages/react-composites/tests/browser/chat/app/FakeAdapterApp.tsx

```diff
@@ -24,9 +24,8 @@ import {
   customOnRenderMessage,
   customOnRenderTypingIndicator
 } from './CustomDataModel';
-import { FakeChatClient } from './fake-back-end/FakeChatClient';
-import { Model } from './fake-back-end/Model';
-import { IChatClient } from './fake-back-end/types';
+// TODO: Should import using path '@internal/fake-backends' from tsconfig but it is not working
```

Comment:
Please fix this TODO before merging.

Reply:
I pushed a commit that _should_ fix the error (I assume) you were seeing in the CI.

Feel free to resolve this comment thread if that succeeds.

Reply:
Thanks for your commits on this branch but I'm still seeing this error locally when running `rushx build:e2e:chat`
![image](https://user-images.githubusercontent.com/79475487/178338236-4fe59e1b-35ee-4db0-9c0a-b4729f29011e.png)

Reply:
Oops. I can't type. Let's have CI try that again.